cypress/integration/navigations.ts

```diff
@@ -0,0 +1,58 @@
+import { IFRAME_SELECTOR } from '../support/constants';
+
+describe('navigations', () => {
+  beforeEach(() => {
+    cy.loadPage('/streamed-epub');
+  });
+
+  it('table of content link should update the page content', () => {
+    cy.iframe(IFRAME_SELECTOR)
+      .findByRole('img', {
+        name: "Alice's Adventures in Wonderland, by Lewis Carroll",
+      })
+      .should('exist');
+
+    // Open TOC menu
+    cy.findByRole('button', { name: 'Table of Contents' }).click();
+
+    // Open chapter 1
+    cy.findByRole('menuitem', { name: 'I: Down the Rab­bit-Hole' }).click();
+    cy.wait(100); // wait for the nested iframe to render
+
+    cy.iframe(IFRAME_SELECTOR)
+      .find('.subtitle')
+      .contains('Down the Rab­bit-Hole');
```

Comment:
I had to explicitly set a wait time for the click event to get registered/ or given some time for the iframe to load the new contents.

Reply:
I think that's to be expected here because we don't get a promise back from R2D2BC letting us know when this is complete, so I think setting a timer is all we can do, unless cypress has a wait to just wait for something to be true (with a given timeout)

Reply:
Although maybe this strategy would be useful: https://www.cypress.io/blog/2020/02/12/working-with-iframes-in-cypress/#slow-loading-frames